conf/prod.routes

```diff
@@ -24,3 +24,8 @@ GET         /email-verification/verification-request                         uk.
 ## Adddress Lookup
 GET         /v2/uk/addresses                                                 uk.gov.hmrc.cgtpropertydisposalsstubs.controllers.AddressLookupController.lookupAddresses(postcode:String)
 
+# Financial Data
+GET        /enterprise/financial-data/:idType/:idNumber/:regimeType     uk.gov.hmrc.cgtpropertydisposalsstubs.controllers.FinancialDataController.getFinancialData(idType: String, idNumber: String, regimeType: String,dateFrom:String,dateTo:String)
+
+# Create Return
+POST        /capital-gains-tax/cgt-reference/:cgtReferenceNumber/return               uk.gov.hmrc.cgtpropertydisposalsstubs.controllers.ReturnController.submitReturn(cgtReferenceNumber:String)
```

Comment:
we already have submit return above (L15)  - I moved all the ETMP stubs to the same section. The financial data one prbably belongs up there as well with the other etmp stub endpoints